src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterVarargs.java

```diff
@@ -11,20 +11,23 @@
 
 public class InputWhitespaceAfterVarargs {
 
-    public void method(int...obj) { // violation
+    public void method(int...obj) { // violation ''...' is not followed by whitespace'
     }
 
-    public InputWhitespaceAfterVarargs(String first, List<Integer>...second) { // violation
+    public InputWhitespaceAfterVarargs(String first,
+           List<Integer>...second) { // violation ''...' is not followed by whitespace'
     }
 
-    public <T> void anotherMethod(List<T>...args) { // violation
+    public <T> void anotherMethod
+            (List<T>...args) { // violation ''...' is not followed by whitespace'
```

Comment:
Why are you wrapping this? Did you try to shorten the method name to fit onto the same line?

Reply:
Done

Reply:
Resolved.